tests/cargo-kani/vecdeque-cve/src/fixed.rs

```diff
@@ -1,3 +1,5 @@
+// From Rust Standard Library. Copyright Rust Developers.
```

Comment:
I don't think rust std uses copyright statements.

Reply:
Artificially added for testing purposes.

Reply:
@fzaiser's generator test cases already had injected copyright notices, so I moved up the licenses them so they test the copyright script. They are merged in main, so I feel it's ok to use.